frontend/src/main.ts

```diff
@@ -4,6 +4,7 @@ import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
 import { ObserveVisibility } from 'vue-observe-visibility';
 import hljs from 'highlight.js';
 import router from './router/index';
+import './styles/highlight-js-style.css';
```

Comment:
Required as the move to Vite somehow broke the existing highlighting mechanism.

Reply:
Seems like the code syntax highlighting is working for me even without this import statement. Could you please check again?